client/dashboard/task-list/tasks/tax.js

```diff
@@ -100,10 +99,14 @@ class Tax extends Component {
 		}
 	}
 
-	isSupportedCountry() {
-		const { countryCode } = this.props;
+	isTaxJarSupported() {
+		const { countryCode, options } = this.props;
 		const { automatedTaxSupportedCountries = [] } = getSetting( 'onboarding', {} );
-		return automatedTaxSupportedCountries.includes( countryCode );
+
+		return (
+			'1' === options.woocommerce_setup_jetpack_opted_in &&
```

Comment:
We should actually be checking `wc_connect_options` for `tos_accepted`.

The Jetpack option actually gets deleted: https://github.com/Automattic/woocommerce-services/blob/62654d48ff51cf119be5c87a455adbbb6e61c44d/woocommerce-services.php#L416-L431

https://github.com/Automattic/woocommerce-services/blob/62654d48ff51cf119be5c87a455adbbb6e61c44d/woocommerce-services.php#L416-L431

Reply:
Oh nice catch!  Updated in https://github.com/woocommerce/woocommerce-admin/pull/3007/commits/2740e0e988d2a0fe52510ff36a1f742f9d880b4f